assets/models/organelles/Flagellum.tscn

```diff
@@ -1,45 +1,47 @@
-shader = ExtResource("2_laoej")
+shader = ExtResource("2_4pd8r")
 shader_parameter/dissolveValue = 0.0
+shader_parameter/dissolveScale = 1.5
+shader_parameter/fresnelValue = 1.0
+shader_parameter/fresnelActive = false
 shader_parameter/outlineWidth = null
 shader_parameter/growColor = null
 shader_parameter/tint = Color(1, 1, 1, 1)
-shader_parameter/jiggleAmount = 0.0005
-shader_parameter/texture = ExtResource("4_w6y0d")
-shader_parameter/dissolveTexture = ExtResource("3_43ujs")
+shader_parameter/jiggleAmount = 0.0
+shader_parameter/texture = ExtResource("4_uwayx")
+shader_parameter/dissolveTexture = ExtResource("3_7rn5o")
 
 [node name="Flagellum" instance=ExtResource("1")]
 
 [node name="Skeleton3D" parent="Armature_001" index="0"]
-bones/0/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
-bones/2/rotation = Quaternion(-0.707107, 2.80375e-23, 2.80375e-23, 0.707107)
-bones/3/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
-bones/4/rotation = Quaternion(-0.707107, 1.1215e-22, 1.1215e-22, 0.707107)
-bones/5/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
-bones/6/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
-bones/7/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
-bones/8/position = Vector3(0.0217572, -0.0374796, 0.266215)
-bones/8/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
-bones/9/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
-bones/11/rotation = Quaternion(-0.707107, -2.80375e-23, -2.80375e-23, 0.707107)
-bones/12/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
-bones/13/rotation = Quaternion(-0.707107, -5.60749e-23, -5.60749e-23, 0.707107)
-bones/14/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
-bones/15/rotation = Quaternion(-0.707107, 0, 0, 0.707107)
+bones/0/position = Vector3(0.018158, -0.0121349, 0.987055)
+bones/1/position = Vector3(-0.077255, -0.057645, -0.06757)
+bones/2/position = Vector3(-0.21558, 0.03514, -0.735139)
+bones/3/position = Vector3(0.10268, 0.071748, -1.40271)
+bones/4/position = Vector3(0.200754, -0.058261, -2.07028)
+bones/5/position = Vector3(-0.187991, -0.070903, -2.73785)
+bones/6/position = Vector3(-0.195265, 0.078756, -3.40542)
+bones/7/position = Vector3(0.229396, 0.061931, -4.07299)
+bones/8/position = Vector3(0.021757, -0.03748, 0.376399)
+bones/9/position = Vector3(-0.196349, -0.029784, -0.375976)
+bones/10/position = Vector3(-0.141033, 0.077108, -1.08831)
+bones/11/position = Vector3(0.240594, 0.021294, -1.73649)
+bones/12/position = Vector3(0.032682, -0.099299, -2.40406)
+bones/13/position = Vector3(-0.308219, 0.000927, -3.07163)
+bones/14/position = Vector3(0.020694, 0.099601, -3.7392)
+bones/15/position = Vector3(0.282216, -0.030774, -4.40677)
 
-[node name="flagella" parent="Armature_001/Skeleton3D" index="0"]
-material_override = SubResource("ShaderMaterial_ff6tk")
+[node name="FlagellumMesh" parent="Armature_001/Skeleton3D" index="0"]
+material_override = SubResource("ShaderMaterial_provs")
 
 [node name="AnimationPlayer" parent="." index="1"]
 autoplay = "swim"
-speed_scale = 0.3
```

Comment:
Why is the default speed scale gone? The intention with that is that the flagellum only plays the full speed swimming animation when the C# code commands it, and otherwise is like in the "idle" state, as there's many more cases where it is useful to have the idle animation than needing to show the max speed.

Reply:
That was unintended. It seemed to work without it, though.

Reply:
I'm 90% sure that the editor would end up displaying the animation playing at 100% speed which looks very busy and not very passive in the editor. So this change would increase visual business in the editor (if I'm right in how I expect this to affect things).